.xo-config.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
> Move the .svglintrc.mjs code body to scripts/svglint-config.mjs. It's related to https://github.com/xojs/xo/issues/673

Can you use `"!.,svglintrc.mjs"` in the `ignores` field of `.xo-config.json`?

Reply:
This won't work. You can try this on your local. This will affect the original XO configuration.

Reply:
I've tested locally and works. See the [`ignores` documentation](https://github.com/xojs/xo?tab=readme-ov-file#ignores), it says:

> Additional ignores can be added here.

Reply:
> I've tested locally and works.

What about other files? XO on other files is not working anymore.

Reply:
Oh I see, what a crazy feature. Let me see if this can be fixed using a *xo-config.mjs* config file and trying to import the rest of the ignore.

Reply:
Sure. Thanks a lot.

Reply:
Okay, seems a bug in the globby dependency of XO.

I think that the most suitable alternative is to add ` --config svglintrc.mjs` to `svglint` script (at *package.json*) move *scripts/svglint-config.mjs* to *svglintrc.mjs* and drop *.svglintrc.mjs*. That will keep the blame of the SVGLint config file, which I'm most concerned about.